### PR TITLE
Support Level 6 (Mega) Raids

### PR DESCRIPTION
**Pull request by georgeherby**
Remove Level 2 & 4 from the dash

### CODE REVIEW

**Comment by kamieniarz** [comment]
No idea why O have this in logs:
```
ERROR 1054 (42S22) at line 1: Unknown column 'raids_6' in 'field list'
```
And this on grafana page:
Bez tytułu

**Comment by na-ji** [comment]
> No idea why O have this in logs:
> 
> ```
> ERROR 1054 (42S22) at line 1: Unknown column 'raids_6' in 'field list'
> ```
> 
> And this on grafana page:
> Bez tytułu

You have to update your database with this file [03_update.sql](https://github.com/muckelba/PogoStats/blob/d3dab93e8266ac0d110285250aa6dcbb717dfa58/sql/03_update.sql) like mentioned in the doc: https://github.com/muckelba/PogoStats#updating

**Comment by muckelba** [comment]
I got one error while testing your PR: `grafana_1   | t=2020-09-04T12:49:54+0000 lvl=eror msg="failed to load dashboard from " logger=provisioning.dashboard type=file name=Default file=/etc/grafana/provisioning/dashboards/default_dashboard.json error="invalid character 'Â' looking for beginning of object key string"` 

But `cat  grafana/dashboards/default_dashboard.json |grep 'Â'` does not return any result. `doos2unix` wasnt a help as well. 

My grafana raid graph only has 5 levels, the sql update worked, i have the new `raids_6` collumn in my DB.

File: grafana/dashboards/default_dashboard.json [hunk 1]
```diff
@@ -563,7 +488,7 @@
         "y": 9
       },
       "hiddenSeries": false,
-      "id": 8,
+      "id": 11,§
```
Comment (na-ji) [inline grafana/dashboards/default_dashboard.json]:
```suggestion
      "id": 11,
```
That's the `Â` char that was causing some errors

Comment (muckelba) [reply grafana/dashboards/default_dashboard.json]: yes, thats it. Thanks!

**Pull request merged**